tools/sdk/include/user_interface.h

```diff
@@ -427,6 +428,17 @@ typedef enum {
     MODEM_SLEEP_T
 } sleep_type_t;
 
+typedef enum {
+    MIN_SLEEP_T,
+    MAX_SLEEP_T
+} sleep_level_t;
+
+bool wifi_set_sleep_level(sleep_level_t level);
+sleep_level_t wifi_get_sleep_level(void);
+
+bool wifi_set_listen_interval(uint8 interval);
```

Comment:
Is there info about what  this listen interval means, and the units of the argument (e.g. ms)?

Reply:
No doc to my knowledge yet.